network/datadog_checks/network/network.py

```diff
@@ -266,8 +266,10 @@ def _submit_devicemetrics(self, iface, vals_by_metric, tags):
             'bytes_rcvd',
             'bytes_sent',
             'packets_in.count',
+            'packets_in.drop',
             'packets_in.error',
             'packets_out.count',
+            'packets_out.drop',
             'packets_out.error',
         ]
```

Comment:
There's a slight problem here - the CI passes because it only runs on systems where you implemented these - linux + windows. However, this function would now fail on BSD/Solaris, because the metrics aren't present there. Could you fix this to account for different platforms?

I think the best thing to do would be creating a method `_get_expected_metrics` which would return list of expected metrics based on platform - the `check` method has a similar piece of code that you could take as an inspiration: https://github.com/DataDog/integrations-core/blob/be05cf5d87e273eb4180d2f7bf19d2a830051056/network/datadog_checks/network/network.py#L107